src/js/components/Skeleton/stories/Group.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+
+import { Box, Button, Card, Grommet, Image, Text, ThemeContext} from 'grommet';
+import { FormAdd } from 'grommet-icons';
+
+const Item = ({ ...rest }) => (
+  <Card
+    width="531px"
+    pad="medium"
+    gap="large"
+    round="medium"
+    flex="grow"
+    border
+    {...rest}
+  >
+    <Box direction="row" gap="large" justify="between" align="center">
+      <Box direction="row" gap="medium">
+        <Box width="96px" height="96px" background="orange!" round="small">
+          <Image />
```

Comment:
<img width="525" alt="Screen Shot 2022-11-08 at 3 00 25 PM" src="https://user-images.githubusercontent.com/42451602/200684595-5cb81bc5-756e-428c-947c-e04711108359.png">

Reply:
Probably. Just need to find something suitable. In the skeleton mode I wouldn't expect the image to have a `src` since that data is probably what you're waiting on

Reply:
good point to me just looked like something was missing